.github/workflows/docs.yaml

```
@@ -48,7 +48,7 @@ jobs:
           environments: docs
           opengl: true
       - name: Build documentation
-        run: pixi run -e docs docs-build
+        run: pixi run -e docs docs-build-parallel
```

```suggestion
        run: pixi run -e docs docs-build
```

Did not work :man_shrugging: